htdocs/admin/emailcollector_list.php

```diff
@@ -29,6 +29,8 @@
 require_once DOL_DOCUMENT_ROOT.'/core/class/html.formcompany.class.php';
 require_once DOL_DOCUMENT_ROOT.'/core/lib/date.lib.php';
 require_once DOL_DOCUMENT_ROOT.'/core/lib/company.lib.php';
+require_once DOL_DOCUMENT_ROOT."/core/class/html.formcategory.class.php";
```

Comment:
No need of $formcategory (a specialiaed class to manage categories). 
You can use $form that is always defined

Reply:
Also fixed in https://github.com/Dolibarr/dolibarr/pull/20669/commits/bdfe7f12d90051e004dc2c9a715d0a7aeb0868c7